src/www/ui/api/index.php

```diff
@@ -138,6 +138,8 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
     $app->patch('/{id:\\d+}', UploadController::class . ':updateUpload');
     $app->put('/{id:\\d+}', UploadController::class . ':moveUpload');
     $app->post('', UploadController::class . ':postUpload');
+    $app->put('/{id:\\d+}/permissions', UploadController::class . ':setUploadPermissions');
+    $app->get('/{id:\\d+}/perm-groups', UploadController::class . ':getGroupsWithPermissions');
```

Comment:
Please add the missing documentation for the endpoint in `src/www/ui/api/documentation/openapi.yaml`

Reply:
Yess, I'll add them in the next PR update.